st2debug/tests/integration/test_submit_debug_info.py

```diff
@@ -121,6 +117,52 @@ def test_create_archive_include_all(self):
         self.assertTrue(os.path.isdir(pack_dir))
         self.assertTrue(not os.path.exists(config_path))
 
+    def test_create_archive_include_all(self):
+        archive_path = create_archive(include_logs=True, include_configs=True,
+                                      include_content=True,
+                                      include_system_info=True)
+        extract_path = tempfile.mkdtemp()
+        self._verify_archive(archive_path=archive_path,
+                             extract_path=extract_path,
+                             required_directories=['logs', 'configs', 'content'])
+
+    def _create_config_yaml_file(self):
```

Comment:
Ah makes sense now. You are right `Travis` or any `CI` specific path would be bad.